src/main/java/Deadline.java

```diff
@@ -0,0 +1,13 @@
+public class Deadline extends Task {
+    protected String by;
+
+    public Deadline(String description, String by) {
+        super(description);
+        this.by = by;
+    }
+
+    @Override
+    public String toString() {
+        return "[D]" + super.toString() + " (by: " + by + ")";
```

Comment:
Perhaps can change the "[D]" to a constant so that other developer knows it is for deadline type. I noticed the same issue in other subclasses too.

Reply:
I think it may not be necessary since it is pretty self explanatory when it is in the deadline class.